style.css

```diff
@@ -0,0 +1,15 @@
+*{
+  background-color: black;
+  color: white;
+  text-align: center;
+}
+header{
+    width: 764px;
+    height: 246px;
+    margin: auto;
+    background-image: url(bus.jpg);
```

Comment:
this errors out.  github is being very helpful when it changes the capitalization to "Bus.jpg"